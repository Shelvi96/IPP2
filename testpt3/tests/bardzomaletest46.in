NEW ID

6372521:4>343365;9
007277>:
80;>7:8
0:28;9121>;486
16;>136:3:116
9:4;50:2>57540534
1332:14>056:58
1;512>;
74;716487;>:286
806:96>258378624:
30070>77:934848
2905;62>521864628:
8:8789:5:6>4975663058
781;>375573564
415369>16:01;5
60:7353>2
@553900:939;72592;88160
@24896:988639
@308:97
@357643252288:955;6174493094:
@;0547;02594
@6067536386053354776;7;76389
@9183232;:9777742:1::0
@10786283;6579;242
@1900442790598:
@36463;9984:772:07;4740
@;9986568144;7388007;;1
@391749768478316;6:75
