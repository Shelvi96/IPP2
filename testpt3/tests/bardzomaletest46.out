7093346227
0
0
46328125979713536
0
22876802611389
301000001
29282
18
3010000010
1000000211
20100001
